GliaWidgets/Sources/EntryWidget/MediaTypeItemStyle.swift

```diff
@@ -90,6 +98,8 @@ extension EntryWidgetStyle {
             secureMessagingMessage: String,
             messageFont: UIFont,
             messageColor: UIColor,
+            unreadMessagesCounterFont: UIFont,
```

Comment:
Should we add also textColor for the counter?

Reply:
Added https://github.com/salemove/ios-sdk-widgets/pull/1112/commits/f99771f3e8f5efd277ab6d66978888c311137bea